baseplate/core.py

```diff
@@ -317,19 +333,44 @@ def __exit__(self, exc_type, value, traceback):
         else:
             self.finish()
 
+    def make_local(self, component_name, op_name):
```

Comment:
i keep dithering on if this should be its own method or just a boolean flag on `make_child`. that'd mean we'd need to figure out a new signature for `make_child` which just returns the new context or something and you have to get at the span from `context.trace`? whatcha think?